src/Components/test/testassets/BasicTestApp/AfterRenderInteropComponent.razor

```diff
@@ -6,8 +6,8 @@
 @functions {
     ElementRef myInput;
 
-    protected override void OnAfterRender()
+    protected override async Task OnAfterRenderAsync()
     {
-        JSRuntime.InvokeAsync<object>("setElementValue", myInput, "Value set after render");
+        await JSRuntime.InvokeAsync<object>("setElementValue", myInput, "Value set after render");
```

Comment:
Out of interest, does it make any difference to the functionality whether the developer returns this task or not? Won't it behave the same if they fire-and-forget the invocation?

I agree your version of this is better, but just want to check whether the old version would be broken now for some reason.

Reply:
Yeah, it has no impact. I was just trying stuff while I was investigating why it didn't work in the first place (until I copied the values)

Reply:
I did undo it to make sure it didn't make any difference.